crates/bevy_ui/src/render/mod.rs

```diff
@@ -170,17 +181,104 @@ pub struct ExtractedUiNodes {
     pub uinodes: EntityHashMap<Entity, ExtractedUiNode>,
 }
 
-pub fn extract_atlas_uinodes(
+pub fn extract_uinode_background(
```

Comment:
```suggestion
pub fn extract_uinode_background_colors(
```

Makes it clearer this system just draws colours not textures.